README.md

```diff
@@ -110,6 +145,7 @@ Required:
 
 Optional:
 
+- `resolve_aliases` (boolean) - If `true`, the post-processor resolves the AWS Systems Manager parameter when the launch template uses it to specify the AMI ID. See [AWS documentation](https://docs.aws.amazon.com/autoscaling/ec2/userguide/using-systems-manager-parameters.html). **Important**: If you set this to `true`, you must add `ssm:GetParameters` permission to the IAM Role.
```

Comment:
Regarding the `ssm:GetParameters` permissions, it would be a good idea to add them to the example IAM role below.

```diff
        "ec2:StopInstances",
        "ec2:TerminateInstances",
+       "ssm:GetParameters" // If "resolve_aliases" is enabled
      ],
```

Reply:
Like it. I will do.

Reply:
Added!